terraform/modules/alertmanager/input.tf

```diff
@@ -2,4 +2,8 @@ variable monitoring_org_name {}
 
 variable monitoring_space_id {}
 
-variable config {}
+variable config { default = "" }
+
+locals {
+  tmp_config   = var.config == "" ? file("${path.module}/config/alertmanger.yml") : file(var.config)
```

Comment:
Can we replace `file(var.config)` with `var.config` to force passing the config as a string?

Reply:
yes